sdks/python/apache_beam/yaml/standard_providers.yaml

```diff
@@ -24,6 +24,29 @@
     version: BEAM_VERSION
   transforms:
      Sql: 'beam:external:java:sql:v1'
+     Flatten: 'beam:schematransform:org.apache.beam:yaml:flatten:v1'
+
+# TODO(robertwb): Auto-detect redundantly provided transforms for maximal fusion.
+- type: 'beamJar'
+  config:
+    gradle_target: 'sdks:java:io:expansion-service:shadowJar'
+    version: BEAM_VERSION
+  transforms:
+     Flatten: 'beam:schematransform:org.apache.beam:yaml:flatten:v1'
+
+- type: 'beamJar'
+  config:
+    gradle_target: 'sdks:java:io:google-cloud-platform:expansion-service:shadowJar'
+    version: BEAM_VERSION
+  transforms:
+     Flatten: 'beam:schematransform:org.apache.beam:yaml:flatten:v1'
+
+- type: 'beamJar'
+  config:
+    gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+    version: BEAM_VERSION
+  transforms:
+     Flatten: 'beam:schematransform:org.apache.beam:yaml:flatten:v1'
```

Comment:
Why are there multiple gradle targets? If it is in multiple, then why the need to specify multiple?

Reply:
The idea here is that we want to be able to use the same environment as proceeding/subsequent operations when possible.

Reply:
Ah ok that makes sense. In that case, should we do the same for other transforms? I assume similar providers (i.e. MapToFields, Filter, etc.) also get packaged in those targets?

Reply:
I'm thinking about how to do this automatically.

Reply:
LGTM in the meantime